### PR TITLE
Includes changes for ignoring background as scenario in scenario count and other minor fixes

### PR DESCRIPTION
**Pull request by spujar**
Includes changes for reporting time in millisecond
Includes changes for conditional checking of feature tag
Includes changes for brand title saying cucumberjs

### CODE REVIEW

File: lib/reporter.js [hunk 1]
```diff
@@ -109,6 +118,11 @@ var generateReport = function(options) {
                 element.ambiguous = 0;
                 element.time = 0;
 
+                if(element.type=="background")
```
Comment (gkushang) [inline lib/reporter.js]: Can you please replace double quotes with single quotes? 
Also the if should be `===` 

```
element.type === 'background'
```

File: templates/bootstrap/index.tmpl [hunk 2]
```diff
@@ -13,7 +13,7 @@
     <div class="navbar navbar-default navbar-static-top" role="navigation">
       <div class="container">
         <div class="navbar-header">
-          <a class="navbar-brand">Cucumberjs Report</a>
+          <a class="navbar-brand"></a>
```
Comment (gkushang) [inline templates/bootstrap/index.tmpl]: I believe we should parameterized the brand `title` instead of completely removing it. If parameter `title` is not passed in the `options` then show the default `Cucumberjs Report` otherwise show the customized title (as a future enhancement). Let me know if you agree.

File: templates/bootstrap/features.tmpl [hunk 3]
```diff
@@ -29,7 +29,7 @@
             <div class="panel panel-default">
               <div class="panel-heading">
                 <h4 class="panel-title">
-                <% var scenarioTagsOnly = feature.tags ? _.drop(element.tags, feature.tags.length) :  element.tags %>
+                <% var scenarioTagsOnly = element.tags %>
```
Comment (gkushang) [inline templates/bootstrap/features.tmpl]: Could we know the reasons for removing the tag filtering? If we remove this then it will pollute the report with duplicate tags at Scenario level. 

Please see below screenshots. 

* The `@unhappy @reporting` are part of the feature file so it should not be at Scenario level. We filter those tags to make it clean

screen shot 2017-04-26 at 10 56 18 am


* With the filtering

screen shot 2017-04-26 at 10 54 28 am

File: templates/bootstrap/index.tmpl (hunk 2 above)
Comment (gkushang) [inline templates/bootstrap/index.tmpl]: For the above comment, I am referring to Cucumber version <2.

Comment (spujar) [inline templates/bootstrap/index.tmpl]: I agree for it to be taken from options if available else default it to Cucumberjs Report

File: lib/reporter.js (hunk 1 above)
Comment (spujar) [inline lib/reporter.js]: Will update as suggested

File: templates/bootstrap/features.tmpl (hunk 3 above)
Comment (spujar) [inline templates/bootstrap/features.tmpl]: Will revert this change as this was more for a specific use where in if the tags are burried under the feature, they weren't visibile unless the feature twiste is expanded.

**Comment by gkushang** [comment]
@spujar thanks for making updates. It looks good to merge apart from missing readMe about `brandtitle` option, will add it before releasing it.

**Comment by ankushmulgir** [comment]
Hello All,
I am also getting the same issue.

Language: VBScript
Can anybody share the script/code for generating duration/time in JSON reporting format( milliseconds)

any inbuilt function or code will be helpful
